tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -423,6 +434,167 @@ RHODS Notification Drawer Should Contain
     Click Element    xpath=//*[contains(@class,'notification-badge')]
     Page Should Contain Element
     ...    xpath=//*[contains(text(),'${message}')]
+    Close Notification Drawer
+
+Open Notebook Images Page
+    [Documentation]    Opens the RHODS dashboard and navigates to the Notebook Images page
+    Page Should Contain    Settings
+    Menu.Navigate To Page    Settings    Notebook Images
+    Wait Until Page Contains    Notebook image settings
+    Page Should Contain    Notebook image settings
+
+Import New Image
+    [Documentation]  software and packages should be lists of lists
+    [Arguments]  ${repo}  ${name}  ${description}  ${software}  ${packages}
+    Sleep  1
+    Open Image Import Popup
+    Input Text    xpath://input[@id="notebook-image-repository-input"]    ${repo}
+    Input Text    xpath://input[@id="notebook-image-name-input"]    ${name}
+    Input Text    xpath://input[@id="notebook-image-description-input"]    ${description}
+    Add Softwares To Custom Image    ${software}
+    Add Packages To Custom Image    ${packages}
+    Click Element    xpath://button[.="Import"]
+
+Open Image Import Popup
+    [Documentation]
```

Comment:
This and other [Documentation] tags should be filled

Reply:
Done